content/data-reporting/reports-api/v3.4/index.mdx

```diff
@@ -289,6 +291,117 @@ https://api.livechatinc.com/v3.4/reports/chats/total_chats \
 </Code>
 </Section>
 
+<Section>
+<Text>
+
+### Ratings
+
+Shows the number of rated chats along with their ratings during a specified period of time.
+
+#### Specifics
+
+|                     |                                                              |
+| ------------------- | ------------------------------------------------------------ |
+| **Method URL**      | `https://api.livechatinc.com/v3.4/reports/chats/ratings`     |
+| **HTTP method**     | POST, GET                                                    |
+| **Required scopes** | -                                                            |
+
+#### Request
+
+See [available parameters and filters](#available-parameters-and-filters).
+
+#### Response
+
+| Field                    | Notes                                                    |
+| ------------------------ | -------------------------------------------------------- |
+| `total`                  | The total number of chats in the specified date range.   |
+| `records`                | Contains the `distribution` objects, for example, `day`. |
+| `records.day.chats`      | The total number of chats that `day`.                    |
+| `records.day.bad`        | The number of chats rated `bad` that `day`.              |
+| `records.day.good`       | The number of chats rated `good` that `day`.             |
+
+</Text>
+<Code>
+<CodeSample path={'REQUEST'}>
+<Sample>
+
+```shell
+curl -X POST \
+https://api.livechatinc.com/v3.4/reports/chats/ratings \
+  -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -d '{
+    "distribution": "day",
+    "filters": {
+      "from": "2021-06-23T00:00:00-00:00",
+      "to": "2021-06-27T23:59:59-00:00"
```

Comment:
can we use the same date range as in other reports?

Reply:
changed, along with response